CMakeLists.txt

```diff
@@ -56,6 +56,21 @@ add_test (NAME qrack_tests
     COMMAND unittest
     )
 
+# Declare the benchmark executable
+add_executable (benchmarks
```

Comment:
If you’ve squashed this into `test`, you shouldn’t need the additional CMake changes.

Reply:
It's still a separate executable, for the moment. We could take about squashing it into the same executable, but there's no duplicated code, right now.